lib/plugins/create/templates/hello-world/serverless.yml

```diff
@@ -21,4 +21,3 @@ functions:
       - http:
           path: hello-world
           method: get
-          cors: true
```

Comment:
@luebken Sure. The main motivation was that the `http` event definition w/o CORS enabled is too basic for "serious" applications.

We also elaborated if we should enable CORS by default whenever you use the `http` event but thought that this might be too much for the first iteration.

More information about the decisions / motivation for the `hello-world` template can be found here: https://github.com/serverless/serverless/pull/3445